sw/tools/opti_dist/NatNetClient.py

```diff
@@ -0,0 +1,438 @@
+﻿import socket
+import struct
+from threading import Thread
+
+def trace( *args ):
+    pass # print( "".join(map(str,args)) )
+
+# Create structs for reading various object types to speed up parsing.
+Vector3 = struct.Struct( '<fff' )
+Quaternion = struct.Struct( '<ffff' )
+FloatValue = struct.Struct( '<f' )
+DoubleValue = struct.Struct( '<d' )
+
+class NatNetClient:

```

Comment:
it seems to be a duplicate of https://github.com/paparazzi/paparazzi/blob/master/sw/ground_segment/python/natnet3.x/NatNetClient.py
Please merge if needed and integrate to current NatNet3 support.

Reply:
I do not know what changed in this version of the NatNetClient, so I won't merge it. I have rewritten the dist.py script to load the client in the natnet3.x folder instead.